modules/gridss/1.0/gridss.smk

```diff
@@ -285,26 +376,27 @@ rule _gridss_viral_annotation:
         viral_ref = str(rules._gridss_input_viral_ref.output), 
         viral_img = str(rules._gridss_setup_viral_ref.output)
     output: 
-        vcf = CFG["dirs"]["viral_annotation"] + "{seq_type}--{genome_build}/{tumour_id}--{normal_id}--{pair_status}/gridss_viral_annotation.vcf.gz" 
+        vcf = temp(CFG["dirs"]["viral_annotation"] + "{seq_type}--{genome_build}/{tumour_id}--{normal_id}--{pair_status}/gridss_viral_annotation.vcf.gz")
```

Comment:
Won't this delete the main output?

Reply:
Fortunately not. In unpaired mode the next step is to filter for only PASS variants and that's the only VCF I want to keep (they can be quite large). For paired mode gripss runs next.